icevision/data/convert_records_to_coco_style.py

```diff
@@ -6,13 +6,104 @@
     "coco_api_from_records",
     "coco_api_from_preds",
     "create_coco_eval",
+    "export_batch_inferences_as_coco_annotations",
 ]
 
 from icevision.imports import *
 from icevision.utils import *
 from icevision.core import *
 from pycocotools.coco import COCO
 from pycocotools.cocoeval import COCOeval
+from icevision.models.inference import *
+import json
+import numpy as np
+import PIL
+
+
+class NpEncoder(json.JSONEncoder):
+    """
+    Makes exporting from IceVision predictions to JSON possible
+    """
+
+    def default(self, obj):
+        if isinstance(obj, np.integer):
+            return int(obj)
+        if isinstance(obj, np.floating):
+            return float(obj)
+        if isinstance(obj, np.ndarray):
+            return obj.tolist()
+        return json.JSONEncoder.default(self, obj)
+
+
+def export_batch_inferences_as_coco_annotations(
+    preds,
+    img_files,
+    transforms,
+    class_map,
+    output_filepath="new_pseudo_labels_for_further_training.json",
+):
```

Comment:
It would be good to add `addl_info` an optional argument to let the user save that info in coco file

We could have: `info=None`  

We could also add `licenses` but maybe it's too much.

Reply:
Good idea